### PR TITLE
Feature/sidebar

### PR DESCRIPTION
**Pull request by jason-herald**
Create sidebar component

### CODE REVIEW

File: packages/ui/src/components/Sidebar/SideHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  active,
+  iconName,
+}: {
+  children: String;
+  active?: Boolean | undefined;
+  iconName?: IconName;
+}) => (
+  <div
+    className={`border-l-4 cursor-pointer select-none text-lg flex font-bold py-1 ${
+      active ? "bg-gray-300 border-gray-600" : "hover:bg-gray-200"
```
Comment (Gayathri-Nakkeeran) [inline packages/ui/src/components/Sidebar/SideHeader.tsx]: Maybe we could change font weight for active and non active headers

File: packages/ui/src/components/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            <div>
+              <img
+                className="h-12 w-12 rounded-full object-cover"
+                src="profile-pic.png"
+                alt="lady"
+              />
+            </div>
+            <div>Sylvia #33457</div>
+          </div>
+          <Button
```
Comment (Gayathri-Nakkeeran) [inline packages/ui/src/components/Sidebar/Sidebar.tsx]: Should be changed once button component is ready

File: apps/web/public/default-pic.png
Comment (Gayathri-Nakkeeran) [inline apps/web/public/default-pic.png:1]: Move images to assets under ui/src

**Review by manojsai-bellamkonda — APPROVED** [review]

**Comment by manojsai-bellamkonda** [comment]
Though the Sidebar component UI is developed, the component yet needs to use dynamic data to render. Should be in the next PR.

**Pull request merged**